main.js

```diff
@@ -175,9 +175,9 @@ define(function (require, exports, module) {
         });
 
         LanguageManager.defineLanguage("latex", {
-            name: "Latex",
+            name: "LaTeX",
             mode: ["stex", "text/x-stex"],
-            fileExtensions: ["tex", "bib", "cls"],
+            fileExtensions: ["tex", "bib", "cls", "ltx", "clo", "sty", "def"],
```

Comment:
what is a .ltx file? Could you point to a spec? Thanks

Reply:
A .ltx file is specifically a LaTeX file, versus just a TeX file, though you can use both.  https://fileinfo.com/extension/ltx#LaTeX_Document talks about the file type.